modules/katamari/src/test/ts/atomic/api/type/TypeTest.ts

```diff
@@ -131,6 +132,21 @@ describe('atomic.katamari.api.struct.TypeTest', () => {
     checkIsFunction(false, 1);
   });
 
+  it('isPromiseLike', () => {
+    checkIsPromiseLike(false, null);
+    checkIsPromiseLike(false, undefined);
+    checkIsPromiseLike(false, true);
+    checkIsPromiseLike(false, false);
+    checkIsPromiseLike(false, 'ball');
+    checkIsPromiseLike(false, {});
+    checkIsPromiseLike(false, objectString);
+    checkIsPromiseLike(false, []);
+    checkIsPromiseLike(false, noop);
+    checkIsPromiseLike(false, [ 1, 3, 4, 5 ]);
+    checkIsPromiseLike(false, 1);
+    checkIsPromiseLike(true, new Promise(noop));
+  });
```

Comment:
Add a couple more tests please

* false for an object with a `then` function
* false for an object with a `catch` function
* true for an object with both `then` and `catch` functions

Reply:
done